api/v1beta1/operatorpolicy_types.go

```diff
@@ -204,14 +204,19 @@ type OperatorPolicyStatus struct {
 	//+optional
 	RelatedObjects []policyv1.RelatedObject `json:"relatedObjects"`
 
-	// The resolved name.namespace of the subscription
+	// The resolved name.namespace of the subscription.
 	ResolvedSubscriptionLabel string `json:"resolvedSubscriptionLabel,omitempty"`
 
+	// The value of the 'operatorpolicy.policy.open-cluster-management.io/namechange' annotation
+	// when the operator name and namespace were most recently changed. This field is used to
+	// prevent some unintentional changes, for example caused by templates.
+	ResolvedSubscriptionLock string `json:"resolvedSubscriptionLock,omitempty"`
```

Comment:
I'm not sure if "lock" or the specifics of this annotation are ideal, I'm open to different names or other suggestions!